src/api-types.ts

```diff
@@ -60,29 +60,30 @@ export const GenerateInputSchema = z.object({
 });
 export type GenerateInput = z.infer<typeof GenerateInputSchema>;
 
-export type GenerateStopReason =
-  | 'NOT_FINISHED'
-  | 'MAX_TOKENS'
-  | 'EOS_TOKEN'
-  | 'CANCELLED'
-  | 'TIME_LIMIT'
-  | 'STOP_SEQUENCE'
-  | 'TOKEN_LIMIT'
-  | 'ERROR';
-
-export interface GenerateResult {
```

Comment:
Backwards incompatible but good riddance. Same for the `Request*` types.

Reply:
This should not be a problem as API types are internal.

Reply:
No they're not, we are exporting them out of the package (on purpose).